lms/djangoapps/courseware/access.py

```diff
@@ -136,6 +140,36 @@ def can_load_forum():
             )
         )
 
+    def can_load_mobile():
+        """
+        Can this user access this course from a mobile device?
+        """
+        return (
+            # check mobile requirements
+            can_load_mobile_not_enrolled() and
+            # check enrollment
+            (
+                CourseEnrollment.is_enrolled(user, course.id) or
+                _has_staff_access_to_descriptor(user, course, course.id)
```

Comment:
The `can_load_mobile` method checks enrollment in addition to the other checks.
The `can_load_mobile_not_enrolled` method does not check enrollment, but checks release date, beta user, and eventually cohorted content (once supported).

How about 'can_load_mobile' and 'can_load_mobile_no_enrollment_check'?